src/frontend/apps/standalone_site/src/hooks/useResponsive.tsx

```diff
@@ -1,12 +1,40 @@
-import { ResponsiveContext } from 'grommet';
-import { useContext } from 'react';
+import { ResponsiveContext, ThemeContext } from 'grommet';
+import { useCallback, useContext } from 'react';
+
+interface Theme {
+  global: {
+    breakpoints: {
+      [key: string]: {
+        value?: number;
+      };
+    };
+  };
+}
 
 export const useResponsive = () => {
   const listSmallerScreen = ['small', 'xsmall', 'xxsmall'];
   const breakpoint = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext) as Theme;
+
+  const isSmallerBreakpoint = useCallback(
```

Comment:
The function is not the same than in your commit message. You should update your commit message accordingly